packages/seacas/libraries/ioss/src/Ioss_ElementTopology.C

```diff
@@ -311,3 +311,25 @@ Ioss::ElementTopology *Ioss::ElementTopology::boundary_type(int bnd_number) cons
   }
   return nullptr;
 }
+
+bool Ioss::ElementTopology::operator==(const Ioss::ElementTopology &rhs)
+{
+  if( this->name_.compare(rhs.name_) != 0 ) { 
```

Comment:
Operator== should not be outputting anything.

Reply:
Addressed in branch `tkordenbrock-seacas/add-ioshell-compare`.